cypress/tests/data/60-content/AmwandengaSubmission.spec.js

```diff
@@ -7,47 +7,256 @@
  *
  */
 
-describe('Data suite tests', function() {
+describe('Data suite: Amwandenga', function() {
 
 	let submission;
 
 	before(function() {
+		const title = 'Signalling Theory Dividends: A Review Of The Literature And Empirical Evidence';
 		submission = {
 			id: 0,
 			section: 'Articles',
 			prefix: '',
-			title: 'Signalling Theory Dividends: A Review Of The Literature And Empirical Evidence',
+			title: title,
 			subtitle: '',
 			abstract: 'The signaling theory suggests that dividends signal future prospects of a firm. However, recent empirical evidence from the US and the Uk does not offer a conclusive evidence on this issue. There are conflicting policy implications among financial economists so much that there is no practical dividend policy guidance to management, existing and potential investors in shareholding. Since corporate investment, financing and distribution decisions are a continuous function of management, the dividend decisions seem to rely on intuitive evaluation.',
-			authors: ['Alan Mwandenga']
+			shortAuthorString: 'Mwandenga, et al.',
+			authorNames: ['Alan Mwandenga', 'Amina Mansour'],
+			assignedAuthorNames: ['Alan Mwandenga'],
+			authors: [
+				{
+					givenName: 'Amina',
+					familyName: 'Mansour',
+					email: 'amansour@mailinator.com',
+					country: 'Barbados',
+					affiliation: 'Public Knowledge Project'
+				}
+			],
+			files: [
+				{
+					'file': 'dummy.pdf',
+					'fileName': title + '.pdf',
+					'mimeType': 'application/pdf',
+					'genre': Cypress.env('defaultGenre')
+				},
+				{
+					'file': 'dummy.xlsx',
```

Comment:
Just philosophically, can we use LibreOffice files?

Reply:
As a counterpoint: shouldn't we be testing with the file formats most commonly used by our community? It probably makes sense for us to have a whole bunch of file formats for the tests, so I could add LibreOffice versions in addition to these.

Reply:
:+1: I've added `.odt` and `.ods` files to the tests. My original plan was to use these to test the document type icons in the UI. But having these formats will be good for us to write mimetype tests, which might break unexpectedly with dependency updates.